app/src/ui/octicons/octicon.tsx

```diff
@@ -40,14 +42,15 @@ export class Octicon extends React.Component<IOcticonProps, {}> {
   private svgRef = createObservableRef<SVGSVGElement>()
 
   public render() {
-    const { symbol, title, tooltipDirection } = this.props
+    const { symbol, title, tooltipDirection, ariaLabel } = this.props
     const viewBox = `0 0 ${symbol.w} ${symbol.h}`
     const className = classNames('octicon', this.props.className)
 
     // Hide the octicon from screen readers when it's only being used
     // as a visual without any attached meaning applicable to users
     // consuming the app through an accessibility interface.
-    const ariaHidden = title === undefined ? 'true' : undefined
+    const ariaHidden =
+      title === undefined && ariaLabel === undefined ? 'true' : undefined
```

Comment:
I am assuming we are not using one variable for this because title can be a `jsx.element`. I tried removing the `jsx.element` and running `build:dev` and didn't get any errors. Thus, I do not believe we are using that functionality. Wondering if we should just use `title` for the aria label? Or alternatively keep what you have, but do something like 
`const ariaLabel = ariaLabel !== undefined ? ariaLabel : title is a string ? title : undefined`.  
^ This does give us the advantage of specifying two different things (visual versus just text).